docs/self-managed/identity/what-is-identity.md

```diff
@@ -5,7 +5,17 @@ sidebar_label: "What is Identity?"
 description: "Identity is the component within the Camunda 8 stack responsible for authentication and authorization."
 ---
 
-Identity is the component within the Camunda 8 stack responsible for authentication and authorization. It allows you to manage:
+Identity is the component within the Camunda 8 stack responsible for authentication and authorization. Identity functions differently in Orchestration clusters and in Console and Web Modeler. For more information on these differences, see the Self-Managed [reference architecture](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster-vs-web-modeler-and-console).
```

Comment:
If we want to start to shift away from the "component" language, we can do that here:

"Identity is responsible for authentication and authorization within the Camunda 8 stack."

Or something similar.